From 8386166cf78deeec61918fcee407c0a95faef9c6 Mon Sep 17 00:00:00 2001
From: Vitaly Kuznetsov <vkuznets@redhat.com>
Date: Fri, 23 May 2014 11:50:07 -0400
Subject: [redhat] RHMAINTAINERS: add another Xen maintainer

Message-id: <1400845807-15990-1-git-send-email-vkuznets@redhat.com>
Patchwork-id: 80340
O-Subject: [RHEL7.1 PATCH] RHMAINTAINERS: List myself among Xen maintainers for RHEL7
Bugzilla:
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>

Add myself as Xen maintainer.

Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 18c4ecd..84818d1 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -711,6 +711,7 @@ F: net/mac80211/
 F: net/wireless/
 
 XEN
+M: Vitaly Kuznetsov <vkuznets@redhat.com>
 M: Radim Krcmar <rkrcmar@redhat.com>
 M: Igor Mammedov <imammedo@redhat.com>
 M: Andrew Jones <drjones@redhat.com>
-- 
1.7.1